src/onedb/fsck/quotas.rb

```diff
@@ -474,11 +474,13 @@ def calculate_vnet_quotas(doc, queries, resource)
             vnet_doc.xpath('VNET/AR_POOL').each do |ar|
                 parent_id = ar.xpath('AR/PARENT_NETWORK_AR_ID')
                 parent_id = parent_id.text unless parent_id.nil?
-                size      = ar.xpath('AR/SIZE').text.to_i
 
-                if !parent_id.nil?
-                    vnet_usage[parent_id] = 0 if vnet_usage[parent_id].nil?
-                    vnet_usage[parent_id] += size
+                next if parent_id.nil? || parent_id.empty?
```

Comment:
Is it correct ? Pass if its nil or its empty, better not (nil or empty) 
next if !(parent_id.nil? || parent_id.empty?)
i think